ci/run_pytests.sh

```diff
@@ -18,15 +18,11 @@ set -e -E -u -o pipefail
 # Support invoking run_cudf_pytests.sh outside the script directory
 cd "$(dirname "$(realpath "${BASH_SOURCE[0]}")")"/../python/tests/
 
-# Unless otherwise specified, use all available GPUs and set
+# Unless `LEGATE_CONFIG` is set, default to all available GPUs and set fbmem/sysmem.
 # LEGATE_TEST=1 to test broadcasting code paths (locally).
-# TODO: Set LEGATE_CONFIG instead (if undefined).  However,
-#       as of 2024-10-11 LEGATE_CONFIG seems broken:
-#       https://github.com/nv-legate/legate.core.internal/issues/1304
+LEGATE_CONFIG=${LEGATE_CONFIG:- --gpus="$(nvidia-smi -L | wc -l) --fbmem=4000 --sysmem=4000"} \
```

Comment:
Admittedly, just part of the PR because the comment contains `24-10`.  But a simple improvement that works now: `LEGATE_CONFIG=` can be used to override settings.